tests/src/rules/first.js

```diff
@@ -35,7 +42,37 @@ ruleTester.run('first', rule, {
                   'use directive';\
                   import { y } from 'bar';"
          , errors: 1
+         , output: "import { x } from 'foo';\
+                  import { y } from 'bar';\
+                  'use directive';"
          })
+  , test({ code: "var foo = bar;\
+                  import { x } from './foo';\
+                  import { y } from './bar';\
+                  import { z } from './baz';"
+         , errors: 3
+         , output: "import { x } from './foo';\
+                  import { y } from './bar';\
+                  import { z } from './baz';\nvar foo = bar;"
```

Comment:
nbd but could this use a continuation instead of a literal `\n`?

Reply:
it must be a seperator here. I thought a line-breaker is suitable, or may be another option?

Reply:
It just looks odd to see the previous lines using \ but this one using \n